angular/projects/spark-angular/src/lib/components/sprk-tooltip/sprk-tooltip.component.ts

```diff
@@ -0,0 +1,237 @@
+import {
+  Component,
+  HostListener,
+  Input,
+  Output,
+  OnInit,
+  ViewChild,
+  EventEmitter
+} from '@angular/core';
+import { uniqueId } from 'lodash';
+
+@Component({
+  selector: 'sprk-tooltip',
+  template: `
+  <span class="sprk-c-Tooltip__container" #containerElement>
+    <button
+      class="sprk-c-Tooltip__trigger"
+      [ngClass]="{
+        'sprk-c-Tooltip__trigger' : true,
+        'sprk-c-Tooltip--toggled' : isOpen
+      }"
+      [attr.aria-expanded]="isOpen ? 'true' : 'false'"
+      [attr.data-analytics]="analyticsString"
+      [attr.data-id]="idString"
+      #triggerElement
+    >
+      <sprk-icon
+        [iconType]="triggerIconType"
+        [additionalClasses]="additionalClassesIcon"
```

Comment:
Hey @bobwise can you update this one to be iconAdditionalClasses. We are standardizing the naming for these. There’s the base additionalClasses and then the other ones should be in the format of elementAdditionalClasses.